src/Materials/Textures/babylon.dynamicTexture.ts

```diff
@@ -178,6 +178,20 @@
             return newTexture;
         }
 
+        /**
+         * Serializes the dynamic texture
+         * @returns a serialized dynamic texture object
+         */
+        public serialize(): any {
+            const serializationObject = super.serialize();

```

Comment:
Does it work on IE?

Reply:
No, this does not work in IE.  I'll work on a fix.

Reply:
A fix or if none found a check to make sure it won't crash